dist/post-events.js

```diff
@@ -0,0 +1,164 @@
+/**
```

Comment:
naming convention for `js` projects in files is camel case: `postEvents.js` (for other js files too).